pages/_app.js

```diff
@@ -1,17 +1,33 @@
 import GlobalStyle from "../styles";
 import initialFlashCards from "../lib.data.json";
-import Header from "@/components/Header/Header";
+import useLocalStorageState from "use-local-storage-state";
+import { nanoid } from "nanoid";
+
+import Layout from "@/components/Layout/Layout";
 
 export default function App({ Component, pageProps }) {
+  const [cards, setCards] = useLocalStorageState("cards", { defaultValue: [] });
   const initialData = initialFlashCards;
-  console.log(initialData);
+  console.log("Klappt", cards);
+
+  function addCard(newCardData) {
+    const newCards = [...cards, { id: nanoid(), ...newCardData }];
```

Comment:
```suggestion
    const newCards = [ { id: nanoid(), ...newCardData }, ...cards];
```

If the newest card is supposed to be always displayed on top, you should turn this around :)